wftests/warrior_tests/wrapper_files/tw_sample.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" ?>
```

Comment:
The directory need to committed under Warriorspace.

Reply:
this will be added in WAR-2066